pkg/route/graph/analysis/analysis.go

```diff
@@ -86,3 +90,25 @@ func FindMissingTLSTerminationType(g osgraph.Graph, f osgraph.Namer) []osgraph.M
 
 	return markers
 }
+
+func FindPathBasedPassthroughRoutes(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
+	markers := []osgraph.Marker{}
+
+	for _, uncastRouteNode := range g.NodesByKind(routegraph.RouteNodeKind) {
+		routeNode := uncastRouteNode.(*routegraph.RouteNode)
+
+		if len(routeNode.Spec.Path) > 0 && routeNode.Spec.TLS != nil && routeNode.Spec.TLS.Termination == routeapi.TLSTerminationPassthrough {
+			markers = append(markers, osgraph.Marker{
+				Node: routeNode,
+
+				Severity: osgraph.ErrorSeverity,
+				Key:      PathBasedPassthroughErr,
+				Message:  fmt.Sprintf("%s is path-based and uses passthrough termination, which is an invalid combination.", f.ResourceName(routeNode)),
+				// TODO: Provide the commands separately for copy/paste
+				Suggestion: osgraph.Suggestion(fmt.Sprintf("Either remove the path or switch the route to use edge termination")),
```

Comment:
ok, updated to suggest the three things that are possible in order to make the route functional.